keyboards/keebio/quefrency/keymaps/GeorgePetri/README.md

```diff
@@ -0,0 +1,48 @@
+# George Petri's Quefrency 65 layout
```

Comment:
Could you rename this so the filename is fully lowercase?

e.g. `keyboards/keebio/quefrency/keymaps/GeorgePetri/readme.md`